docs/Deploying-Pega-on-AKS.md

```diff
@@ -409,6 +410,55 @@ Use the provided example addons.yaml file to configure the use of an Application
 
 3. Save the file.
 
+
+### Updating the backingservices.yaml Helm chart values (Supported when installing or upgrading to Pega Infinity 8.6 and later)
+
+
+Adding the Pega configuration files to your Helm installation on your local system
+----------------------------------------------------------------------------------
+Pega maintains a repository of Helm charts that are required to deploy Pega Platform and related resources using Helm, including a generic version of the following charts. After you add the repository to your local system, you can customize these configuration files for your backing service deployment:
+
+- pega/backingservices - Use this chart to set customization parameters for your deployment. You will modify this chart later in the deployment tasks.
+
+1. To add the backingservices repository to your Helm installation, enter:
+
+    `$ helm repo add pega https://dl.bintray.com/pegasystems/pega-helm-charts`
```

Comment:
Do you want this to be updated to latest repo `helm repo add pega https://pegasystems.github.io/pega-helm-charts` instead of bintray as per latest [README ](https://github.com/pegasystems/pega-helm-charts/blob/master/README.md#getting-started)
 I see this repo in couple of other README's in this PR.

Reply:
good call @MadhuriArugula - last week I worked with JohnP, RaghuS, and RagsS to merge changes into the main readme and four runbooks so they point to the appropriate site where the repos are now hosted. bintray is no longer supported so these docs are correct:
https://github.com/pegasystems/pega-helm-charts/blob/master/README.md
line 33

https://github.com/pegasystems/pega-helm-charts/blob/master/docs/Deploying-Pega-on-AKS.md
line 447

https://github.com/pegasystems/pega-helm-charts/blob/master/docs/Deploying-Pega-on-EKS.md
line 411

https://github.com/pegasystems/pega-helm-charts/blob/master/docs/Deploying-Pega-on-GKE.md
line 212

https://github.com/pegasystems/pega-helm-charts/edit/master/docs/Deploying-Pega-on-PKS.md
line 171

Reply:
@MadhuriArugula I have corrected the url references in latest commit, please verify. Thank you.

Reply:
looks good to me Vinay...thank you!

Reply:
@pega-kshev LGTM, approved!